src/main/java/com/jamieswhiteshirt/clothesline/common/item/ClotheslineItems.java

```diff
@@ -35,7 +35,7 @@ public boolean connect(ItemUsageContext from, ItemUsageContext to) {
             if (world.getBlockState(to.getBlockPos()).getBlock() == ClotheslineBlocks.CLOTHESLINE_ANCHOR) {
                 Vec3d fromVec = Utility.midVec(from.getBlockPos());
                 Vec3d toVec = Utility.midVec(to.getBlockPos());
-                BlockHitResult hitResult = world.rayTrace(new RayTraceContext(fromVec, toVec, RayTraceContext.ShapeType.COLLIDER, RayTraceContext.FluidHandling.NONE, to.getPlayer()));
+                BlockHitResult hitResult = world.raycats(new RaycastContext(fromVec, toVec, RaycastContext.ShapeType.COLLIDER, RaycastContext.FluidHandling.NONE, to.getPlayer()));
```

Comment:
`raycats`? This is either a mapping bug if you didn't see if this compiles.

Reply:
that would be a typo i didnt catch, i had to redo my changes on github because i accidentally downloaded from your repo not my fork

Reply:
just fixed the typo. i have tested compile locally

Reply:
just tested the api, all functions appear to work. all blocks and items work, adding and removing from them works